source/core/types/ut_event_listener.tpb

```diff
@@ -80,15 +78,20 @@ create or replace type body ut_event_listener is
           self.reporters(i).after_calling_after_each(treat(a_item as ut_test));
         elsif a_event_name = ut_utils.gc_after_all then
           self.reporters(i).after_calling_after_all(treat(a_item as ut_logical_suite));
-        else
-          raise_application_error(ut_utils.gc_invalid_rep_event_name,'Invalid reporting event name - '|| nvl(a_event_name,'NULL'));
         end if;
-      else
-        raise_application_error(ut_utils.gc_invalid_rep_event_time,'Invalid reporting event time - '|| nvl(a_event_timing,'NULL'));
       end if;
     end loop;
 
   end fire_event;
 
+  overriding member procedure fire_event(self in out nocopy ut_event_listener, a_event_name varchar2) is
```

Comment:
I propose to call it `fire on event` to be consistent with other similar procedures